randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -1404,9 +1404,9 @@ Extra - asset_id: collision_camera_019
           Any of the following:
               Cross Bomb
               # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
-              Bomb and Power Bombs
+              Bomb and Main Power Bomb and Power Bombs
```

Comment:
I would totally just use the Lay Power Bomb Template once there are two items involved.